### ISSUE TITLE
Mario Tennis video playback is broken, game crashes

### ISSUE THREAD

**Issue by bronka13** [cemu-project/Cemu#1332]
### Current Behavior

The video on the menus of the game aren't playing in the latest version, which causes the game to crash. I reverted back to 2.0 to get it back working.

### Expected Behavior

Videos on title screen, game mode selection, court selection should play normally.

### Steps to Reproduce

Run the game and get to main screen.

### System Info (Optional)

OS: Windows 10
GPU: Radeon RX 570

**Comment by Kdash4** [comment]
I second this. Newest updates broke the game.  No video playback and game crashes randomly. On 2.0 builds it was working without issue for me

**Comment by petran79** [comment]
I notice that game crashes after exiting the match and returning to the menu,

**Comment by pblue3** [comment]
> I second this. Newest updates broke the game. No video playback and game crashes randomly. On 2.0 builds it was working without issue for me

I confirm, here too the game crashes randomly.

**Comment by callo90** [comment]
Same here, I disabled the VSync and the Graphic packs and stopped crashing for me,

**Comment by Hexaae** [comment]
... you also mean main menu background stays all black? Same bug?

**Comment by Hexaae** [comment]
FYI: latest version without this bug was 2.0-88 (Experimental). It started with 2.0-89 (Experimental)

**Comment by qqlow** [comment]
yes
i reverted the commit and built from source, now running with usual cemu bugs but no more crashes

**Comment by eduararley** [comment]
I can confirm that reverting to 2.0-88 (Experimental) this issue is fixed and game doesn't crash anymore; although there are other bugs like missing icons in in-game action messages

**Comment by mrdc** [comment]
Crashes for me when I use Vulkan, but works OK (brief test) when using experimental Metal backend.

**Comment by YodiSki** [comment]
Still crashing on 2.6 has anyone found a fix for this
Using Vulkan tried OpenGL and turning off graphics packs and still crashes in the menus when selecting gamemode
I quickly went into a game and it played the game fine but as soon as i got back into the menu it crashes

**Comment by qqlow** [comment]
as i said that commit is the issue, vulkan or opengl or gfx has nothing to do with this

**Comment by Hexaae** [comment]
@qqlow 
did you send a pull request on GitHub?

**Comment by qqlow** [comment]
no, trying to debug this, is very low priority for me, already full time on other stuff so just reverting and building did it for me, cemu dev @Exzap should handle this, which is also the author of that commit
